test/integration/puma/plugin_test.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+require "test_helper"
+
+class PumaPluginTest < ActiveSupport::TestCase
+  self.use_transactional_tests = false
+
+  setup do
+    cmd = %w[
+      bundle exec puma
+        -b tcp://127.0.0.1:9222
+        -C test/dummy/config/puma.rb
+        --dir test/dummy
+        -s
+        config.ru
+    ]
+
+    @pid = fork do
+      exec(*cmd)
+    end
+  end
+
+  teardown do
+    Process.kill 'INT', @pid
```

Comment:
A super minor nit, just to match other usages in the codebase:
```suggestion
    Process.kill :INT, @pid
```

Reply:
fixed!